### ISSUE TITLE
Refiner for Android SDK crashing : lateinit property koinApp has not been initialized

### ISSUE THREAD

**Issue by anandsahi** [refiner-io/mobile-sdk-android#2]
**Prerequisite :**
link to refiner survey : https://app.refiner.io/projects/95973fc0-0381-11ee-962a-65025d48f380/overview

(1) **Refiner SDK Initialises successfully :** 
log : Refiner: Refiner SDK is initialized successfully! 1.2.0 
(2) Trigger event inside targeting tab is set to **Manual**
(3) Survey should open every time user clicks on a button
(4) Recurrence is set to **Repeating**
(5) Follow up mode is disabled

However, upon calling the showForm method using manual launch trigger, the application crashing on the action

`btnShowSurvey.setOnClickListener {
            Refiner.showForm(formUuid = FORM_ID, force = true)    //causing crash, form id is not null here.
  }`

**Complete error log** : 
`java.lang.RuntimeException: Unable to start activity ComponentInfo{com.ezetap.service.demo/io.refiner.ui.RefinerSurveyActivity}: kotlin.UninitializedPropertyAccessException: **lateinit property koinApp has not been initialized**
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2666)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2733)
        at android.app.ActivityThread.-wrap12(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1480)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6153)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
     Caused by: kotlin.UninitializedPropertyAccessException: lateinit property koinApp has not been initialized`
![Screenshot from 2023-07-04 18-08-49](https://github.com/refiner-io/mobile-sdk-android/assets/138589795/70fa21e6-421d-4c10-8021-34466107339b)

**Comment by tekinalper** [comment]
Are you getting this crash through our sample app or your app? If you didn't use our sample app, can you try to reproduce the case with it? I just want to be sure if the integration is done the same as in the document. 

By the way, please use the latest version 1.2.2

**Comment by anandsahi** [comment]
@tekinalper Using the latest version 1.2.2. The crash happens when this form id is used - `a97aa510-0381-11ee-965f-052a6a98e2bf` Used the same form id in sample app, it is not crashing. 

It crashes only if we set the trigger event to "Manual" and update the survey alias, it doesn't crash if we set the trigger event to "Immediately, Time delay, etc"

The setup has been done as mentioned : 
1. Initializing the Refiner SDK, got success message -> I/Refiner: Refiner SDK is initialized successfully! 1.2.2
2. Refiner.identifyUser()
3. calling Refiner.showForm()

form id - a97aa510-0381-11ee-965f-052a6a98e2bf
project id - 95973fc0-0381-11ee-962a-65025d48f380
